build/int.cloudbuild.yaml

```
@@ -104,6 +104,28 @@ steps:
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do destroy budget-local']
 
+# essential contacts
+- id: go-init-essential-contacts-example
```

nit: since we are making this the standard
```suggestion
- id: init-essential-contacts-example
```